package.json

```diff
@@ -54,7 +54,12 @@
     "tsc": "tsc",
     "lint": "npm-run-all -p lint:*",
     "lint:ts": "tslint -p . --format verbose",
-    "lint:sass": "sass-lint -v"
+    "lint:sass": "sass-lint -v",
+    "start:examples": "npm run start -- --examples",
+    "e2e": "start-server-and-test start:examples http://localhost:9090/health/ready cy:run",
+    "e2e:dev": "start-server-and-test start:examples http://localhost:9090/health/ready cy:open",
+    "cy:run": "cypress run",
+    "cy:open": "cypress open"
```

Comment:
Do you really need this scripts?

Reply:
I thought so, but I could just quote commands in `start-server-and-test`!